eZ/Bundle/EzPublishCoreBundle/EventListener/ViewControllerListener.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
 
     public static function getSubscribedEvents()
     {
-        return array( KernelEvents::CONTROLLER => 'getController' );
+        return [KernelEvents::CONTROLLER => ['getController', 10]];
```

Comment:
@yannickroger Please fix the array notation and merge the priority commit into your main one.